### DISCUSSION TITLE
What is "corium"?

### DISCUSSION THREAD

**Discussion by iexa** [reloadware/reloadium#45] [Q&A] [answered]
I wanted to check how this package works but I see it imports a precompiled reloadium.corium module which is at the moment available for macos, linux and windows systems between python 3.6- 3.10

There is no information available about this "corium" at all. Is that part proprietary, or what is the reason for that?

Thanks,

**Comment by dkrystki** [comment, marked as answer]
Hi @iexa , Yes, corium is a package that is currently closed sourced. 
I explained my decision here https://github.com/reloadware/reloadium/discussions/35
The rest of the project is fully open source.
